### PR TITLE
New unlocked paywall story

### PR DESCRIPTION
**Pull request by cellog**
# Description

This adds a new storybook story for the unlocked paywall. Currently it is blank, but when #980 is implemented, it will show the paywall flag.

screen shot 2019-01-19 at 5 29 44 pm
screen shot 2019-01-19 at 5 29 40 pm

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: unlock-app/src/stories/creator/Paywall.stories.js [hunk 1]
```diff
@@ -4,38 +4,80 @@ import { storiesOf } from '@storybook/react'
 import Paywall from '../../pages/paywall'
 import createUnlockStore from '../../createUnlockStore'
 
-const lock = {
-  address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
-  keyPrice: '10000000000000000000',
-  expirationDuration: '86400',
-  maxNumberOfKeys: 800,
-  outstandingKeys: 32,
+const myLock = {
```
Comment (julien51) [inline unlock-app/src/stories/creator/Paywall.stories.js]: Why the change to `myLock`? Is that necessary for this PR?

Comment (cellog) [reply unlock-app/src/stories/creator/Paywall.stories.js]: this was a mystery to me too, but I just realized I copied it from a unit test that uses myLock instead of lock. I can change it back :)
